flink-ml-python/pyflink/ml/core/param.py

```diff
@@ -86,6 +87,8 @@ def get_param_map(self) -> Dict['Param[Any]', Any]:
     @staticmethod
     def _is_compatible_type(param: 'Param[V]', value: V) -> bool:
         if value is not None and param.type != type(value):
+            if type(value).__name__ == 'DenseVector' or type(value).__name__ == 'SparseVector':
+                return issubclass(type(value), param.type)
```

Comment:
Could you please illustrate why we need to add this code here, and why we don't need to add corresponding codes in the Java implementation?

Reply:
In the Java code, using `param.clazz.isAssignableFrom(value.getClass())` to check the compatible. 
But in the python code, type and class are different.